### ISSUE TITLE
Possibly incorrect or outdated Studio quickstart PyTorch tutorial

### ISSUE THREAD

**Issue by carmat88** [scaleoutsystems/examples#10]
**Scenario:** This Studio [quickstart tutorial](https://github.com/scaleoutsystems/examples/tree/main/tutorials/studio/quickstart#pytorch), cannot retrieve the archived model pointed within the `vgg11_scripted.mar` file via `git-lfs`

**Problem:** Once cloned the [examples repo](https://github.com/scaleoutsystems/examples) and cd into the [quickstart folder](https://github.com/scaleoutsystems/examples/tree/main/tutorials/studio/quickstart), running the command `git-lfs pull `results in the following error:
```
carmat@work:quickstart$ git-lfs pull
[756ed4e37232a5862a7ae7b68ef0077d6dc3cfa187bd596e9916b7aded9ccc27] Object does not exist on the server: [404] Object does not exist on the server                                                                                                                                                                
error: failed to fetch some objects from 'https://github.com/scaleoutsystems/examples.git/info/lfs'
```
**Solution:** Updating tutorial with the a new `vgg11_scripted.mar` file.

**Comment by carmat88** [comment]
Hey @ahellander! It seems that you have moved all the files for this quick tutorial folder a month ago. Do you know which archived model this `vgg11_scripted.mar` is supposed to point at? Many thanks!

**Comment by ahellander** [comment]
Yes, it was moved from here:  https://github.com/scaleoutsystems/test-examples

**Comment by carmat88** [comment]
> Yes, it was moved from here: https://github.com/scaleoutsystems/test-examples

Thank you, I'll check if it works from there then.

**Comment by carmat88** [comment]
**For future reference:** when moving files from one repository to another, it is then necessary to setup again the git lfs tracking for big files, otherwise the reference and the hash of such files get broken.
